packages/stryker/README.md

```diff
@@ -87,22 +87,22 @@ This is the same globbing format you might know from [Grunt](https://github.com/
 
 You can *ignore* files by adding an exclamation mark (`!`) at the start of an expression.
 
-#### Files to mutate
-**Command line:** `[--mutate|-m] src/**/*.js,a.js`
-**Config file:** `mutate: ['src/**/*.js', 'a.js']`  
-**Default value:** *none*  
-**Mandatory**: No  
-**Description:**  
+### `mutate` [`string[]`]
+
+Default: *none*  
+Command line: `[--mutate|-m] src/**/*.js,a.js`
+Config file: `mutate: ['src/**/*.js', 'a.js']`
```

Comment:
I'm missing a new line here (you can see it if you preview the markdown file:
![image](https://user-images.githubusercontent.com/4221997/48116312-00a2a200-e266-11e8-8d5d-cbe7537029be.png)

Reply:
Done - I think this might have been something with my editor formatting...